docker-compose.yml

```
@@ -384,6 +384,12 @@ services:
         <<: *depends_on-healthy
       web:
         <<: *depends_on-healthy
+  vroom:
+    <<: *restart_policy
+    image: "VROOM_IMAGE"
```

```suggestion
    image: "$VROOM_IMAGE"
```

Should be inferred from variable